tests/common/db/packaging.py

```diff
@@ -41,8 +39,8 @@ class ProjectFactory(WarehouseFactory):
     class Meta:
         model = Project
 
-    id = factory.LazyFunction(uuid.uuid4)
-    name = factory.fuzzy.FuzzyText(length=12)
+    id = factory.Faker("uuid4", cast_to=None)
+    name = factory.Faker("pystr", max_chars=12)
```

Comment:
Looks like this is failing test due to a name collision. Probably needs to have a better unique handler.